### DISCUSSION TITLE
Possible way to wipe everything and start Fresh

### DISCUSSION THREAD

**Discussion by tcochran1987** [nextcloud/all-in-one#5754] [Questions] [answered]
Something odd happened when I was trying to setup the AIO Master Container, which is now causing things to be able to start fully. I am wondering if there is a way to completely wipe out all existing settings and start the whole process over again?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-properly-reset-the-instance
